### ISSUE TITLE
TextAreaRow does not hide placeholder when value is setted

### ISSUE THREAD

**Issue by prine** [xmartlabs/Eureka#12]
First of all thanks for this awesome library!

I am not 100% sure if I am doing it right. But when I set a value on a TextAreaRow the placeholder is still beeing shown and a strange overlay with the value is also displayed.

![screen shot 2015-10-02 at 13 45 48](https://cloud.githubusercontent.com/assets/265674/10245861/07619762-690c-11e5-8f40-3ab9227dbfe9.png)

Here is the code I use to set the value form the TextAreaRow:
```swift
<<< TextAreaRow("textEN") {
    $0.placeholder = "Text EN"
    $0.value = self.report?.textEN
}
```

**Comment by mtnbarreto** [comment]
@prine 
We have already fixed the issue, take a look here #1

**Comment by prine** [comment]
Ah sorry, didnt see that there was already an issue about that. Would be nice to have these fixes integrated in the Cocoapods version.
